### PR TITLE
order status update is done

### PR DESCRIPTION
**Pull request by zahirzohair**

### CODE REVIEW

File: resources/views/shopOwner/order.blade.php [hunk 1]
```diff
@@ -1,30 +1,73 @@
...
+                
+                 <td>{{$order->id}}</td>
+                <td>{{$order->customer->name}}</td>
+                <td>{{$order->total_price}}</td>
+                <td><form action="{{route('main-siteorderEdit',[ app('request')->route('subdomain') ?? '', $order->id ]) }}" class="btn btn-primary" method="POST">
+                  {{ csrf_field() }}
+                  <input type="hidden" name="_method" value="PATCH" />
+                    <select name="status">
+                          @if($order->status == 1)
+                              <option selected>Unpaid</option>
```
Comment (bjoe88) [inline resources/views/shopOwner/order.blade.php:32]: This will make the options appeal double. And when selecting this option. It will not have value.

Comment (bjoe88) [reply resources/views/shopOwner/order.blade.php:32]: See https://github.com/z-solution/kiahStore/blob/master/resources/views/shop/productList.blade.php#L9-L12 how to do it

Comment (zahirzohair) [reply resources/views/shopOwner/order.blade.php:32]: the if and else conditions are just for displaying the status based on database value. it does not show double

Comment (bjoe88) [reply resources/views/shopOwner/order.blade.php:32]: Not sure what you mean. 
let look at this example. If the $order->status = 4. Line 40 will be rendered. And line 54 also. You will have 2 options for shipping.

**Pull request merged**